android/src/app/organicmaps/settings/SettingsPrefsFragment.java

```diff
@@ -241,11 +241,11 @@ public void onViewCreated(View view, @Nullable Bundle savedInstanceState)
   {
     super.onViewCreated(view, savedInstanceState);
     mPreferenceScreen = getPreferenceScreen();
-    mStoragePref = getPreference(getString(R.string.pref_storage));
+    mStoragePref = findPreference(getString(R.string.pref_storage));
```

Comment:
I couldn't reproduce case when `findPreference(getString(R.string.pref_storage))` returns null. This setting is never removed at runtime.